### PR TITLE
L2-432: Disburse Neuron UI

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

User can open the Disburse Neuron Modal and select a destination account.

# Changes

* New DisburseNeuronModal modal component.
* New Component TransactionInfo to share UI between steps of different wizards.

# Tests

* New test for DisburseNeuronModal.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
Only UI in this PR. No disburse functionality.


https://user-images.githubusercontent.com/98735375/164242720-210b1132-0dcf-4028-afda-6e62ca9686ce.mov

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis @peterpeterparker please review

File: frontend/svelte/src/lib/modals/neurons/DisburseNeuronModal.svelte [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  };
+</script>
+
+<WizardModal {steps} bind:currentStep bind:this={modal} on:nnsClose>
+  <svelte:fragment slot="title"
+    ><span data-tid="disburse-neuron-modal">{currentStep?.title}</span
+    ></svelte:fragment
+  >
+  {#if currentStep.name === "SelectDestination"}
+    <Address bind:address on:submit={onEnterAddress} />
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/modals/neurons/DisburseNeuronModal.svelte:66]: can you create dedicated component for the two steps?

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/modals/neurons/DisburseNeuronModal.svelte:66]: sure

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker ready for another review

File: frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte:1]: can you add a TODO / task to refactor this component and `NewTransactionDestination`

They are quite similar

**Review by peterpeterparker — COMMENTED** [review]
small comments

File: frontend/svelte/src/tests/lib/modals/neurons/DisburseNeuronModal.spec.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    const { container, queryByTestId } = await renderDisburseModal(mockNeuron);
+
+    const addressInput = container.querySelector("input[type='text']");
+    expect(addressInput).not.toBeNull();
+    const continueButton = queryByTestId("address-submit-button");
+    expect(continueButton).not.toBeNull();
+    expect(continueButton?.getAttribute("disabled")).not.toBeNull();
+
+    const address =
+      "cd70bfa0f092c38a0ff8643d4617219761eb61d199b15418c0b1114d59e30f8e";
```
Comment (mstrasinskis) [inline frontend/svelte/src/tests/lib/modals/neurons/DisburseNeuronModal.spec.ts:70]: Is it possible to replace the real address w/ some dummy?

**Review by mstrasinskis — APPROVED** [review]
LGTM. One question regarding test data.

File: frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte (hunk 2 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte:1]: ok

File: frontend/svelte/src/tests/lib/modals/neurons/DisburseNeuronModal.spec.ts (hunk 3 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/tests/lib/modals/neurons/DisburseNeuronModal.spec.ts:70]: I used our mockAccount

File: frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte (hunk 2 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/neuron-detail/SelectDestinationAccount.svelte:1]: done

**Review by peterpeterparker — APPROVED** [review]
Thanks

**Pull request merged**